gio/Gir.toml

```diff
@@ -997,6 +1002,34 @@ status = "generate"
     name = "set_child_setup"
     cfg_condition = "unix"
 
+[[object]]
+name = "Gio.Task"
+status = "generate"
+    [[object.function]]
+    name = "return_error"
+    #wrong mutability
+    ignore = true
+    [[object.function]]
+    name = "get_priority"
+    #wrong priority type
+    ignore = true
+    [[object.function]]
+    name = "set_priority"
+    #wrong priority type
+    ignore = true
+    [[object.function]]
+    name = "propagate_boolean"
+    #wrong return value
```

Comment:
This should probably also be manually implemented

Reply:
This is already manually implemented in `gio/src/task.rs` (actually ... because it was the easiest to do :)) I will implement `propagate_value` and `return_value` as well